src/application/state.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+
+class ApplicationState:
+    """
+    Application state.
+
+    For the simplicity, we store this as a global, but better would be to
+    instantiate this as an object and pass at around.
+    """
+
+    show_status_bar = True
+    current_path = None
+
+    @staticmethod
```

Comment:
Why is this static?

Reply:
so that methods from `ThoughtBox` can call them without having to initalize it?

Reply:
But then what is the point of a getter and setter without having a persistent object?

Reply:
Yeah other than this fix this is great! I guess the appstate can be initialized then take as a param to ThoughtBox because the dialogs take ThoughtBox.